content/Lab1/70_Cleanup/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Cleanup"
+weight: 70
+---
+
+[Setup Cleanup - Destroy created resources](../70_Cleanup/1_cleanup.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description